src/config/generatorsGlobal.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright (c) 2019, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/**
+ * ./src/config/generatorsGlobal.js
```

Comment:
I think this would be clearer if it was called generatorTokens.

Reply:
@jgraff2 I created this file for anything which needs sharing among different generators. Naming this 'generatorTokens' would make it specific only for tokens. Do you think thats better?

Reply:
Can you think of anything else we would need to share? The OAuth token token should be the only thing that's generator-specific that isn't included in the generator object that comes from Refocus.

Reply:
Hmm.. I think it could be any authentication detail (like tokens, client Ids, passwords etc), not just tokens - but I agree I cant think of anything except authentication details. So, call it _generatorAuth_ or something similar?